spec/services/total_express/status_reader_spec.rb

```diff
@@ -51,6 +51,14 @@
   after { Timecop.return }
 
   describe '#parse' do
+    context 'when order have more than a return in api' do
```

Comment:
me parece que faz mais sentido se for:

```suggestion
    context 'when response have multiple orders' do
```

Reply:
:+1: